src/Shared/Microsoft.DotNet.GitHub.Authentication.Tests/GitHubAppTokenProviderTests.cs

```diff
@@ -43,7 +43,7 @@ public void AppTokenReturnsValue()
         {
             // Just use a random RSA key, since we just want to see if it can produce... something
             // There isn't a great way to use a "test" key, so random is what we'll use
-            using var value = RSA.Create(4096);
+            using var value = RSA.Create(4096); // lgtm [cs/cryptography/default-rsa-key-construction] False positive. This does not use the default constructor.
```

Comment:
This seems like a very basic false positive. Are we in contact with the team responsible for this tool so they can update it? I don't really like having comments to appease bad tooling in the code.

Reply:
Agree on all of that. I'm already searching for a contact. I intend to undo this once a better solution is available.